tls/s2n_cipher_preferences.c

```diff
@@ -921,9 +921,9 @@ const struct s2n_cipher_preferences cipher_preferences_cloudfront_tls_1_2_2018 =
 
 struct s2n_cipher_suite *cipher_suites_cloudfront_tls_1_2_2019[] = {
     &s2n_ecdhe_rsa_with_aes_128_gcm_sha256,
+    &s2n_ecdhe_rsa_with_aes_128_cbc_sha256,
```

Comment:
should this get to moved below s2n_ecdhe_rsa_with_aes_256_gcm_sha384 to prefer gcm ciphers?

Reply:
Yes, the correct order should be:
&s2n_ecdhe_rsa_with_aes_256_gcm_sha384,
&s2n_ecdhe_rsa_with_aes_128_gcm_sha256,
&s2n_ecdhe_rsa_with_aes_256_cbc_sha384
&s2n_ecdhe_rsa_with_aes_128_cbc_sha256